src/status_im/ui/screens/profile/events.cljs

```diff
@@ -1,125 +1,69 @@
 (ns status-im.ui.screens.profile.events
-  (:require [clojure.spec.alpha :as spec]
-            [re-frame.core :as re-frame]
-            [status-im.ui.components.react :as react]
-            [status-im.chat.constants :as chat-const]
+  (:require [re-frame.core :as re-frame]
             [status-im.ui.screens.profile.navigation]
-            [status-im.ui.screens.accounts.utils :as accounts.utils]
-            [status-im.chat.events :as chat-events]
-            [status-im.chat.commands.core :as commands]
             [status-im.utils.handlers :as handlers]
-            [status-im.utils.handlers-macro :as handlers-macro]
-            [status-im.utils.image-processing :refer [img->base64]]
-            [taoensso.timbre :as log]))
+            [status-im.ui.screens.profile.models :as models]))
```

Comment:
I would call this `models.profile` or at least add `profile` in the name, and also I would not add `-profile` to the methods in the namespace, so in the namespace you have `(defn save ...)` and when using it outside you have: `(models.profile/save ...)`.

Basically I subscribe to these guidlines https://leanpub.com/elementsofclojure

But just a minor point